.github/workflows/test.yaml

```diff
@@ -48,5 +48,5 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: ./coverage.out
-          name: dms-api
+          name: catalyst-api
```

Comment:
this would apply only when the project gets renamed :|

try setting a `CODEOWNERS` file and see if that grants you access to the project :smile:

Reply:
Yeah, I was going to wait for the rename before merging